e2e/app/page/workspace-edit-page.ts

```diff
@@ -223,7 +223,7 @@ export const FIELD = {
       type: 'radiobutton',
       textOption: {textContains: LABEL_ALIAS.YES_FOCUS_ON_UNDERREPRESENTED_POPULATION}
     },
-    noOnUnderrepresentedPopulationRadiobutton: {
+    noUnderrepresentedPopulationRadiobutton: {
```

Comment:
what does this change do?

Reply:
typo. ui text string does not contain `on` word.

Reply:
ok